src/pages/patientView/trialMatch/TrialMatchTable.tsx

```diff
@@ -335,6 +339,17 @@ export default class TrialMatchTable extends React.Component<ITrialMatchProps> {
                     title="OncoKB Matched Trials General Feedback"
                     userEmailAddress={AppConfig.serverConfig.user_email_address}
                 />
+                {   this.selectedTrialFeedbackFormData &&
+                    <TrialMatchFeedback
+                        show={!!this.selectedTrialFeedbackFormData}
```

Comment:
don't need show prop and internal logic anymore.

Reply:
I added the logic here because the window always popup the Exit Confirmation modal.
![Screen Shot 2019-11-19 at 6 04 58 PM](https://user-images.githubusercontent.com/14971266/69194549-63747d80-0af7-11ea-9fc5-8ab9480b45e4.png)